docs/proposals/resize-when-creating-by-group.md

```diff
@@ -0,0 +1,127 @@
+## Summary
+
+Expand the PVC's initial capacity based on the largest capacity in specified PVCs.
+
+### Motivation
+
+One use case for pvc-autoresizer is automatically extending a PV/PVC for multiple Pod/PV pairings, such as a MySQL cluster.
+In such a cluster, if a Pod/PV fails and a PV/PVC is rebuilt for the restore process, a PV is created based on the PVC template. Even if a PV of the same size as the others is actually needed, a PV of the template's capacity is created, resulting in a case where the restore process stops due to insufficient capacity.
+
+To solve this problem, we are planning to add a new function to pvc-autoresizer to create a PV/PVC with sufficient capacity at the time of creation, when a PVC of the same group already exists.
+
+### Goals
+
+- Provide the ability to expand the PVC's initial capacity based on the largest capacity in specified PVCs.
+- The PVC for determining capacity should be user-selectable.
+- [moco](https://github.com/cybozu-go/moco) checks the PV capacity at the beginning of the restore process. It should be able to handle such cases.
```

Comment:
```suggestion
- [MOCO](https://github.com/cybozu-go/moco) checks the PV capacity at the beginning of the restore process. It should be able to handle such cases.
```

By the way, I think the proposal should be general, and should not be too specific to MOCO, even though our motivation comes from it. It might be better to remove this item from the goals.

Reply:
@llamerada-jp 
> By the way, I think the proposal should be general, and should not be too specific to MOCO, even though our motivation comes from it. It might be better to remove this item from the goals.

How do you think about my comment above?

Reply:
@peng225 
You are correct. I change the sentence of `Goals` and `Decision outcome`. Would you please check again?